server/models/rpaTaskModel.js

```diff
@@ -1,21 +1,22 @@
 const mongoose = require('mongoose');
 
-const {Schema} = mongoose;
+const { Schema } = mongoose;
 
 const rpaVariableSchema = new Schema({
-    Name: String,
-    Type: String,
-    Required: Boolean,
-    InfoText: String 
+  name: String,
+  type: String,
+  required: Boolean,
+  infoText: String,
+  index: Number
 });
 
 const rpaTaskSchema = new Schema({
   Application: String,
   Task: String,
   Code: String,
+  outputValue: Boolean,
   InputVars: [rpaVariableSchema],
   Output: rpaVariableSchema
 });
 
-mongoose.model('rpa-tasks-variable', rpaVariableSchema);
```

Comment:
why was this deleted?

Reply:
It is not necessary, as we are never referencing that Schema as a model